packages/react-data-grid-addons/src/menu/ContextMenu.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { ContextMenu } from 'react-contextmenu';
 
-export default function ReactDataGridContextMenu({ children }: { children: React.ReactNode }) {
-  return <ContextMenu id="reactDataGridContextMenu">{children}</ContextMenu>;
+export default function DataGridContextMenu({ children }: { children: React.ReactNode }) {
+  return <ContextMenu id="dataGridContextMenu">{children}</ContextMenu>;
```

Comment:
This seems a break change, maybe for `id` we keep it?

Reply:
Reverted.